scania-digital-design-system/tegel
# fix(textarea): updating disable label color

## **Describe pull-request**  
The label color for text area was incorrect and have in this PR been updated according to the [Figma design](https://www.figma.com/design/d8bTgEx7h694MSesi2CTLF/Tegel-UI-Library?node-id=34279-126066&node-type=frame&t=OMmW16UPWFXCIjYE-0).

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3621](https://tegel.atlassian.net/browse/CDEP-3621)    

## **How to test**  
**Test 1**
1. Go to the aws preview link
2. Navigate to the textarea component
3. Add an label outside the textarea and toggle disabled.
4. Now verify that the color is correct according to the [Figma Design System](https://www.figma.com/design/d8bTgEx7h694MSesi2CTLF/Tegel-UI-Library?node-id=34279-126066&node-type=frame&t=OMmW16UPWFXCIjYE-0)

**Test 2**
1. Add an label **inside** the textarea and toggle disabled.
2. Now verify that the color is correct according to the [Figma Design System](https://www.figma.com/design/d8bTgEx7h694MSesi2CTLF/Tegel-UI-Library?node-id=34279-126066&node-type=frame&t=OMmW16UPWFXCIjYE-0)

## **Checklist before submission**

## **Suggested test steps**

## **Screenshots**  

<img width="979" alt="outside-pr-fix" src="https://github.com/user-attachments/assets/47fba036-ecbb-4c49-9ec9-fb83ef8b2805">
<img width="993" alt="inside-pr-fix" src="https://github.com/user-attachments/assets/447efe50-3f45-4307-a434-c57ae14a66a5">
<img width="1015" alt="outside-production" src="https://github.com/user-attachments/assets/6582bc68-5fd9-4024-9ade-3116b9bfe894">
<img width="1052" alt="inside-produciton" src="https://github.com/user-attachments/assets/d845c174-df71-41c4-a65f-dbf10b064f6c">


[CDEP-3621]: https://tegel.atlassian.net/browse/CDEP-3621?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
